content/api/trafiklab-apis/netex-regional/siri.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Siri
```

Comment:
Including "realtime data" might be clearer for users not familiar with the SIRI name.

e.g. "Realtime data (SIRI)"